extensions/fluent-outdoors/src/TrackItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { ActionPanel, List, Action, Icon, Color, showToast, Toast } from "@raycast/api";
+import { useState } from "react";
+import { getFavouriteTracksStorage } from "./Storage";
+import { TrackDetails } from "./TrackDetails";
+import { OnFavouriteTracksUpdateAction, Track } from "./types/common";
+import { getTrackCondition } from "./utils";
+
+export function TrackItem({
+  track,
+  isFavourite = false,
+  onFavouriteTracksUpdate,
+}: {
+  track: Track;
+  icon?: Icon;
+  color?: Color;
+  isFavourite?: boolean;
+  onFavouriteTracksUpdate: OnFavouriteTracksUpdateAction;
+}) {
+  const [favourite, setFavourite] = useState(isFavourite);
+  const storage = getFavouriteTracksStorage();
+  let trackIcon = Icon.Snowflake;
+  let trackColor = Color.PrimaryText;
+  let trackActionText = "Mark as favourite";
```

Comment:
```suggestion
  let trackActionText = "Mark as Favourite";
```

Reply:
Suggestion along with the icon added into changeset [3ae4507](https://github.com/raycast/extensions/pull/4474/commits/3ae4507f9cd72c2b80cac1f8a78f489565fb3179)